integration/deployment_test.sh

```diff
@@ -20,34 +20,49 @@ CC_VERS=0
 FAILURES=0
 
 run_test() {
-    try ${PEER_CMD} chaincode install -l fpc-c -n auction_test -v ${CC_VERS} -p examples/auction/_build/lib
 
-    try ${PEER_CMD} chaincode install -l golang -n example02 -v ${CC_VERS} -p github.com/hyperledger/fabric/examples/chaincode/go/example02/cmd
-
-    try ${PEER_CMD} chaincode instantiate -o ${ORDERER_ADDR} -C ${CHAN_ID} -n auction_test -v ${CC_VERS} -c '{"Args":["My Auction"]}' -V ecc-vscc
-    sleep 3
+    # install examples/auction
+    PKG=/tmp/auction_test.tar.gz
+    try ${PEER_CMD} lifecycle chaincode package --lang fpc-c --label auction_test --path ${FPC_TOP_DIR}/examples/auction/_build/lib/ ${PKG}
+    try ${PEER_CMD} lifecycle chaincode install ${PKG}
+    PKG_ID=$(${PEER_CMD} lifecycle chaincode queryinstalled | awk "/Package ID: auction_test/{print}" | sed -n 's/^Package ID: //; s/, Label:.*$//;p')
+    try ${PEER_CMD} lifecycle chaincode approveformyorg -C ${CHAN_ID} --package-id ${PKG_ID} --name auction_test --version ${CC_VERS}
+    try ${PEER_CMD} lifecycle chaincode checkcommitreadiness -C ${CHAN_ID} --name auction_test --version ${CC_VERS}
+    try ${PEER_CMD} lifecycle chaincode commit -C ${CHAN_ID} --name auction_test --version ${CC_VERS}
+
+    # install something non-fpc
+    PKG=/tmp/marbles02.tar.gz
+    try ${PEER_CMD} lifecycle chaincode package --lang golang --label example02 --path github.com/hyperledger/fabric-samples/chaincode/marbles02/go ${PKG}
+    try ${PEER_CMD} lifecycle chaincode install ${PKG}
+    PKG_ID=$(${PEER_CMD} lifecycle chaincode queryinstalled | awk "/Package ID: example02/{print}" | sed -n 's/^Package ID: //; s/, Label:.*$//;p')
+    try ${PEER_CMD} lifecycle chaincode approveformyorg -C ${CHAN_ID} --package-id ${PKG_ID} --name example02 --version ${CC_VERS}
```

Comment:
Minor thing but given that we use now marbles02, we might name the chaincode also marbles02 rather than examples02 (which was the code name of the old chaincode)?